index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+      <div class="main-div">
+          <section class="header-section">
+              <header class="toolbar">
+                  <a href="#">My Logo</a>
+                  <nav>
+                    <img src="image/toggle.png" alt="Toggle" />
```

Comment:
Please be sure to fix this requirement properly. As you can see in the following image, your hamburger button is not showing 😥 Its most likely due to a path issue with the image you are referencing 🤓